Document issue with installing profiler with older versions of GCC (#14765)
DataDog/documentation: content/en/tracing/profiler/profiler_troubleshooting.md

Without this flag, profiles for short-lived Resque jobs will be unavailable.

> ## Profiling does not turn on due to "Your ddtrace installation is missing support for the Continuous Profiler because compilation of the Ruby VM just-in-time header failed. Your C compiler or Ruby VM just-in-time compiler seem to be broken."

Reviewer: ```suggestion
## Profiling does not turn on due to "missing support" error
```

> There is a known incompatibility between Ruby 2.7 and older GCC versions (4.8 and below) that impacts the profiler ([upstream Ruby report][6], [`dd-trace-rb` bug report][7]).

Reviewer: ```suggestion
There is a known incompatibility between Ruby 2.7 and older GCC versions (4.8 and below) that impacts the profiler ([upstream Ruby report][6], [`dd-trace-rb` bug report][7]). This can result in the following error message: "Your ddtrace installation is missing support for the Continuous Profiler because compilation of the Ruby VM just-in-time header failed. Your C compiler or Ruby VM just-in-time compiler seem to be broken."

```

> To fix this, we recommend updating your operating system/docker image or GCC version.

Reviewer: ```suggestion
To fix this, update your operating system, Docker image, or GCC version to something more recent than v4.8.
```

> For further help with this issue, please contact support and include the output of running `DD_PROFILING_FAIL_INSTALL_IF_MISSING_EXTENSION=true gem install ddtrace`, as well as the resulting `mkmf.log` file.

Reviewer: ```suggestion
For further help with this issue, [contact support][2] and include the output of running `DD_PROFILING_FAIL_INSTALL_IF_MISSING_EXTENSION=true gem install ddtrace` and the resulting `mkmf.log` file.
```

Without this flag, profiles for short-lived Resque jobs will be unavailable.

> ## Profiling does not turn on due to "missing support" error

Reviewer: ```suggestion
## Profiling does not turn on because compilation of the Ruby VM just-in-time header failed
```

There is a known incompatibility between Ruby 2.7 and older GCC versions (4.8 and below) that impacts the profiler ([upstream Ruby report][6], [`dd-trace-rb` bug report][7]). This can result in the following error message: "Your ddtrace installation is missing support for the Continuous Profiler because compilation of the Ruby VM just-in-time header failed. Your C compiler or Ruby VM just-in-time compiler seem to be broken."

> To fix this, update your operating system, Docker image, or GCC version to something more recent than v4.8.

Author: On a second look at this, is it just me or does this sentence read like you can update the operating system or docker image to "v4.8" whereas the "v4.8" only refers to the GCC version?

Reviewer: ```suggestion
To fix this, update your operating system or Docker image so that the GCC version is something more recent than v4.8.
```
